### PR TITLE
Spiffs bug496

### PR DESCRIPTION
**Pull request by geeksville**
for your consideration and merging @mc-hamster  (see my comments on my last commit when reviewing ;-))

### CODE REVIEW

File: bin/build-all.sh [hunk 1]
```diff
@@ -87,8 +87,12 @@ platformio lib update
 do_boards "$BOARDS_ESP32" "false"
 do_boards "$BOARDS_NRF52" "true"
 
+echo "Building SPIFFS for ESP32 targets"
+pio run --environment tbeam -t buildfs
+cp .pio/build/tbeam/spiffs.bin $OUTDIR/bins/universal/spiffs-$VERSION.bin
```
Comment (mc-hamster) [inline bin/build-all.sh:92]: I don't believe the ESPHome Flasher tool we document as a way to install from GUI "[Installing from a GUI - Windows and Mac](https://github.com/meshtastic/Meshtastic-device)" supports writing to the spiffs area of the flash. The alternative may be the ESPRESSIF [flash download tool](https://www.espressif.com/en/products/socs/esp32/resources), but that only works for Windows.

Comment (geeksville) [reply bin/build-all.sh:92]: hmm good point and a possible problem for adoption.  I guess the good news is: if the user uses esphome, the next time they update the app over bluetooth the android app **will** write the spiffs partition.

Comment (mc-hamster) [reply bin/build-all.sh:92]: device-update.sh and device-install.sh will also need to be updated.

Check this out (long term solution): 

https://github.com/doayee/esptool-esp32-gui

Shouldn't be a stretch to first add the ability to flash the spiffs location and then eventually make it a standalone tool to pull the latest Meshtastic firmware binaries form a CDN and upload it to the device.

Comment (mc-hamster) [reply bin/build-all.sh:92]: Now that I look at how that application is built, should be able to interface directly with esptool and build a gui without all the cruft of that esptool-esp32-gui. (I'm not volunteering for this, yet)

**Pull request merged**